bogo/config.json

```diff
@@ -380,8 +380,10 @@
   },
   "TestLocalErrorMap": {
     "SendServerHelloAsHelloRetryRequest": "remote error: error decoding message",
-    "GarbageCertificate-Server-TLS12": "remote error: access denied",
-    "GarbageCertificate-Server-TLS13": "remote error: access denied",
+    "GarbageCertificate-Server-TLS12": "remote error: bad certificate",
```

Comment:
Please splice the changes from this commit into the earlier changes that require these changes to make the tests pass, so that each commit can pass the tests.